rapthor/lib/miscellaneous.py

```diff
@@ -349,6 +349,31 @@ def delete_directory(dirname):
             raise e
 
 
+def ra2hhmmss(deg):
+    """Convert RA coordinate (in degrees) to HH MM SS"""
+
+    from math import modf
```

Comment:
I prefer to have all imports at the top of the file. IIRC that's also in the PEP-8 style guide.

Reply:
Yep, me too! I moved it.